scripts/validate-markdown-values

```diff
@@ -3,12 +3,13 @@
 set -e -o pipefail -u
 
 self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+root_dir=$(cd $PWD &>/dev/null; pwd -P)
 source "$self_dir"/utils.sh
+source "$root_dir"/intellijVersions.properties
```

Comment:
Although... in principle, I'd be careful, as syntax of .properties files is much _dumber_... many things that are just plain strings in .properties will be interpreted differently by Bash:

```properties
never_execute_this_command=$(rm -rf --no-preserve-root /)
```

So still, I'd use `grep -Po` with a lookbehind (see below + https://virtuslab.github.io/tips/#regex/lookbehind-lookahead) to extract the value instead of sourcing the properties file!